### ISSUE TITLE
Ant Status Does Not Update in Getting Started After Manual Set

### ISSUE THREAD

**Issue by JoelProminic** [prominic/Moonshine-IDE#544]
We noticed this behavior with #493:
1. Clear Ant Home in the Moonshine Settings
2. Remove or rename MoonshineSDKs/Ant if it exists
3. Open Help > Getting Started.  Ant shows as not found
4. Use the gear and configure a valid value for Ant Home.  See #493 for an example using Homebrew
5. Save the settings.  BUG:  The icon does not update for Apache Ant
6. Close and reopen Getting Started.  The Ant icon is updated.

This may trigger for other SDKs or OSes as well.

**Comment by rat-moonshine** [comment]
The behavior is true for all other SDKs as well, as modifying through Settings doesn't update any opened 'Getting Started' tab immediately at this moment. I'll see how easily I can able to do it.

**Comment by rat-moonshine** [comment]
Based on usage in both MSDKI and Moonshine, the SDK Installer library marks its items based on different things. This also does some additional works when work inside Moonshine. 

Once Getting Started tab opened, removing any path from Moonshine Settings not immediately affect the Getting Started view, thus we introduced a refresh icon at top-right corner. Once clicked, this suppose to do a check-only and update the checkmark to the items in the view.

However, an item may still show check-marked because of following reasons:
1. Path exists in environment-variable (Windows)
2. Path exists in default download location

Moonshine do not sets Moonshine Settings' path by clicking on this refresh-icon but when the Getting Started tab re-opened. In different scenario setting up the settings' path may not be an intentional effect when clicking on the refresh-icon.

I have automated this UI refresh process to Getting Started tab now after a Save happens to the Moonshine Settings (which was doing by refresh-icon until now).
